pages/index.tsx

```diff
@@ -1,75 +1,347 @@
+    icon: VideoCameraIcon,
+  },
+  {
+    name: '4K Support',
+    description:
+      'Request standard and 4K content separately using multiple Radarr/Sonarr servers.',
+    image: (
+      <img
+        src="/icons/4k.svg"
+        className="w-6 h-6 text-white"
+        aria-hidden="true"
+      />
+    ),
+  },
+  {
+    name: 'Plex Scan',
+    description:
+      'Overseerr scans your Plex library at an interval, so we know what is already available in your libraries.',
+    image: (
+      <img
+        src="/icons/plex.svg"
+        className="w-6 h-6 text-white"
+        aria-hidden="true"
+      />
+    ),
+  },
+  {
+    name: 'Notification Agents',
+    description:
+      'Several notification agents are directly supported, such as Discord, Slack, Email, and others.',
+    icon: BellIcon,
+  },
+  {
+    name: 'Mobile Friendly',
+    description:
+      'Use Overseerr as a near-native mobile app by adding it to your homescreen. The entire app is designed to be used on any screen size.',
+    icon: DeviceMobileIcon,
+  },
+  {
+    name: 'Quotas',
+    description:
+      'Set limits to how many movies or seasons a user can request per a configurable amount of days.',
+    icon: AdjustmentsIcon,
+  },
+];
+
+interface FAQItem {
+  id: number;
+  question: React.ReactNode;
+  answer: React.ReactNode;
+}
+
+const faqs: FAQItem[] = [
+  {
+    id: 1,
+    question: 'Is there support for other languages?',
+    answer:
+      'Overseerr has already localized into over 16+ languages thanks to the community! If your language has not been localized yet, we are always accepting new translations!',
+  },
+  {
+    id: 2,
+    question: 'Is there support for local users?',
+    answer:
+      'You can create local users alongside Plex OAuth-based users. You can use this feature to give access to Overseerr to your Plex home users.',
+  },
+  {
+    id: 3,
+    question: 'Does Overseerr support the new Plex Movie/TV agents?',
+    answer:
+      'Yes! We have full support for both the new Plex Movie Agent and the new Plex TV Agent!',
+  },
+  {
+    id: 4,
+    question: 'Does Overseerr support anime agents?',
+    answer: (
+      <span>
+        Overseerr currently has full support for{' '}
+        <a
+          href="https://github.com/ZeroQI/Hama.bundle"
+          target="_blank"
+          rel="noreferrer"
+          className="text-white underline"
+        >
+          HAMA
+        </a>
+        .
+      </span>
+    ),
+  },
+  {
+    id: 5,
+    question: 'Can I use multiple Radarr/Sonarr servers?',
+    answer:
+      'You can add as many of them as you like! You can switch between servers using the Advanced Requester when approving requests. We also support 4K Radarr/Sonarr servers.',
+  },
+  {
+    id: 6,
+    question: 'Can I import my Plex Users?',
+    answer:
+      'Yes, of course! By default, any Plex account with access to your Plex server can log in without being imported!',
```

Comment:
```suggestion
    question: 'Can I import my Plex users?',
    answer:
      'Yes, of course! Any Plex account with access to your Plex server can log in without being imported.',
```
(I do think we should add a setting/option to allow/disable this though.  And _then_ we'd be able to say that "by default" this is the case.)